opennms-webapp/src/main/java/org/opennms/web/admin/discovery/ActionDiscoveryServlet.java

```diff
@@ -244,9 +244,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             LOG.debug("Adding Exclude Range");
             String ipAddrBegin = request.getParameter("erbegin");
             String ipAddrEnd = request.getParameter("erend");
+            String location = request.getParameter("erlocation");
             ExcludeRange newER = new ExcludeRange();
             newER.setBegin(ipAddrBegin);
             newER.setEnd(ipAddrEnd);
+            if(location!=null && !"".equals(location.trim()) &&
```

Comment:
Can we simplify, or explain this conditional? Would be nice to add a utility function to `LocationUtils` for this type of check.